Examples/OneSignalDemo/app/src/main/java/com/onesignal/sdktest/ui/CustomAlertDialogBuilder.java

```diff
@@ -0,0 +1,195 @@
+package com.onesignal.sdktest.ui;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.view.KeyEvent;
+import android.view.View;
+import android.widget.Button;
+
+import com.onesignal.sdktest.R;
+
+public class CustomAlertDialogBuilder extends AlertDialog.Builder {
+
+    /*
+     * Click listener
+     */
+    private DialogInterface.OnClickListener mPositiveButtonListener = null;
+    private DialogInterface.OnClickListener mNegativeButtonListener = null;
+    private DialogInterface.OnClickListener mNeutralButtonListener = null;
+    private DialogInterface.OnDismissListener mOnDismissListener = null;
+
+    /*
+     * Buttons text
+     */
+    private CharSequence mPositiveButtonText = null;
+    private CharSequence mNegativeButtonText = null;
+    private CharSequence mNeutralButtonText = null;
+
+
+    /*
+     * Buttons
+     */
+    private Button positiveButton;
+    private Button negativeButton;
+
+    private View view;
+    private Boolean mCancelOnTouchOutside = null;
+    private boolean isCancelable = true;
+
+    public CustomAlertDialogBuilder(Context context, View view) {
+        super(context);
+        this.view = view;
+    }
+
+    public CustomAlertDialogBuilder setOnDismissListener (DialogInterface.OnDismissListener listener) {
```

Comment:
Remove space after `setOnDismissListener`

Reply:
Got it